syntax = "proto3";
package streetgangpb;


message CreateWorldResponse
{
	string track_id = 1;
	int32 result = 2;
    uint64 world_id = 3;
	string world_name = 4;
}
